packages/pages/src/components/analytics/provider.tsx

```diff
@@ -22,29 +22,29 @@ export function AnalyticsProvider(
     enableTrackingCookie,
     enableDebugging,
     templateData,
-  } = props;
+  } = props
 
-  const analyticsRef = useRef<AnalyticsMethods | null>(null);
+  const analyticsRef = useRef<AnalyticsMethods | null>(null)
 
   if (analyticsRef.current === null) {
-    analyticsRef.current = new Analytics(templateData, requireOptIn);
+    analyticsRef.current = new Analytics(templateData, requireOptIn)
```

Comment:
did y'all update the linter or something to remove semicolons? this can be unsafe in certain situations so we've always leaned towards requiring them.